drivers/renesas/rcar/console/rcar_printf.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2015-2017, Renesas Electronics Corporation. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef	RCAR_PRINTF_H__
+#define	RCAR_PRINTF_H__
+
+#include <string.h>
+
+void rcar_set_log_time(void);
```

Comment:
This function is not called from anywhere and its declaration should be removed.